From 684bb98b31d338d85e1e6089cac381a507a5b4d8 Mon Sep 17 00:00:00 2001
From: Luigi Toscano <luigi.toscano@tiscali.it>
Date: Fri, 13 Jun 2014 02:41:50 +0200
Subject: [PATCH 06/12] Do not set global loading of DTD and entities, no more
 needed

The global settings have been replaced by the fine-grained
parameters passed to xmlReadFile.
Moreover a libxml2 regression prevents those parameters from
being used when the patch for CVE-2014-0191 is applied, see
https://bugzilla.gnome.org/show_bug.cgi?id=730290
A new libxml2 patch is going to be deployed soon, but anyway
this code works even without setting those parameters.

CCBUG: 335001
---
 kdoctools/meinproc.cpp | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/kdoctools/meinproc.cpp b/kdoctools/meinproc.cpp
index 0467f22..de4fbc8 100644
--- a/kdoctools/meinproc.cpp
+++ b/kdoctools/meinproc.cpp
@@ -157,9 +157,6 @@ int main(int argc, char **argv) {
         }
     }
 
-    xmlSubstituteEntitiesDefault(1);
-    xmlLoadExtDtdDefaultValue = 1;
-
     QVector<const char *> params;
 #ifndef Q_WS_WIN
     // libxslt parses the path given to outputFile as XPath expression which fails
-- 
1.8.3.1

